### PR TITLE
SickChill: Use included Python in DSM7

### PR DESCRIPTION
**Pull request by tunisiano187**
_Motivation:_  Explain here what the reason for the pull request is.
_Linked issues:_  Optionally, add links to existing issues or other PR's

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [ ] Package upgrade completed successfully
- [ ] New installation of package completed successfully

### CODE REVIEW

File: spk/sickchill/Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ SPK_REV = 1
 SPK_ICON = src/sickchill.png
 
 DEPENDS = cross/$(SPK_NAME)
-SPK_DEPENDS = "python3>=3.6:git"
+SPK_DEPENDS = "git"
```
Comment (hgy59) [inline spk/sickchill/Makefile:7]: You need at least make this dependent whether the package is created for DSM 7.

Comment (tunisiano187) [reply spk/sickchill/Makefile:7]: Even if it's part of DSM 7 ?

Comment (hgy59) [reply spk/sickchill/Makefile:7]:
```suggestion
include ../../mk/spksrc.common.mk
ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
SPK_DEPENDS = "git"
else
SPK_DEPENDS = "python3>=3.6;git"
endif
```

Comment (BenjV) [reply spk/sickchill/Makefile:7]: @tunisiano187
The python package of the SynoCommunity is different package then the python package included in DSM.
You can install both of them on a Nas.

Comment (hgy59) [reply spk/sickchill/Makefile:7]: > 
> 
> @tunisiano187
> The python package of the SynoCommunity is different package then the python package included in DSM.
> You can have both of them on a Nas.
> 
> Anything changed in the SynoCommunity package will not effect the Synology package at all.

Yes, but @tunisiano wants to use the included python on DSM 7 and not depend on synocommunity package AIUI.

Comment (BenjV) [reply spk/sickchill/Makefile:7]: My misunderstood, my mistake!

Comment (tunisiano187) [reply spk/sickchill/Makefile:9]: Thank you for the informations.
Yes the main goal is that.

**Review by publicarray — APPROVED** [review]

File: spk/sickchill/Makefile (hunk 1 above)
Comment (BKSteve) [reply spk/sickchill/Makefile:9]: if you do this and use the native python in DSM7 how does this affect the `sickchill/src/service-setup.sh` which contains `PYTHON_BIN="/var/packages/python38/target/bin"` in the first line?

**Pull request closed without merging**